site/content/docs/docker/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "ZAP Docker Documentation"
+type: page
+layout: links
```

Comment:
I'd be tempted to add a short paragraph introducing the docker images - eg they are an easy way to run ZAP, especially in a CI/CD environment.